include/linux/df-detection.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __LINUX_DOUBLE_FETCH_DETECTION_H__
+#define __LINUX_DOUBLE_FETCH_DETECTION_H__
+#ifdef CONFIG_DF_DETECTION
+#include <linux/types.h>
+#include <linux/stackdepot.h>
+#define DF_INIT_SIZE 16
+#define DF_MAX_RECORDS 1024
+#define MAX_LEN 1 << 20
+#define STACK_DEPTH 64
+#define DFETCH_ENABLE			_IO('c', 254)
+#define DFETCH_DISABLE			_IO('c', 255)
+#define BYTE_MAX 256
+struct df_address_range{
```

Comment:
Please add a space between the struct name and the opening brace here and below.
Consider running clang-format to fix such nits altogether.